src/test/migrations/steps/20250129053400-step-add-cycle_source_uuid-column.ts

```diff
@@ -0,0 +1,44 @@
+import { Objects } from 'utils/objects'
+
+import { CycleName } from 'meta/assessment'
+
+import { AssessmentController } from 'server/controller/assessment'
+import { BaseProtocol, DB } from 'server/db'
+
+const mapping: Record<CycleName, CycleName> = {
+  '2025': '2020',
+  latest: '2025',
+}
+
+export default async (client: BaseProtocol) => {
+  await DB.query(`
+      alter table public.assessment_cycle
+          add cycle_uuid_source uuid
+              constraint assessment_cycle_assessment_cycle_source_fk
+                  references public.assessment_cycle (uuid)
+                  on update cascade on delete set null;
+  `)
+
+  const assessments = await AssessmentController.getAll({}, client)
+  await Promise.all(
+    assessments.map((assessment) =>
```

Comment:
Not sure if using `client (tx)` and `DB` mixed can cause problems?
`client` is tx outside of `DB`

Reply:
Exactly what I meant, should use `DB` for all or `DB.tx` after `DB.query`